From 7003ae2245e4738631f5ee9829098282f14d3d81 Mon Sep 17 00:00:00 2001
From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Sat, 2 Jan 2021 21:08:59 +0100
Subject: [PATCH 0473/2201] overlays: give Seeed Studio CAN BUS FD HAT a -v2
 postfix

There are several versions of the Seeed Studio CAN BUS FD HAT. This is the
second version, based on the mcp2518fd, so give it a -v2 postfix.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 arch/arm/boot/dts/overlays/Makefile                       | 2 +-
 arch/arm/boot/dts/overlays/README                         | 8 ++++----
 ...fd-hat-overlay.dts => seeed-can-fd-hat-v2-overlay.dts} | 0
 3 files changed, 5 insertions(+), 5 deletions(-)
 rename arch/arm/boot/dts/overlays/{seeed-can-fd-hat-overlay.dts => seeed-can-fd-hat-v2-overlay.dts} (100%)

diff --git a/arch/arm/boot/dts/overlays/Makefile b/arch/arm/boot/dts/overlays/Makefile
index 667c489bfef3..3e05cb399323 100644
--- a/arch/arm/boot/dts/overlays/Makefile
+++ b/arch/arm/boot/dts/overlays/Makefile
@@ -161,7 +161,7 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	sc16is752-spi1.dtbo \
 	sdhost.dtbo \
 	sdio.dtbo \
-	seeed-can-fd-hat.dtbo \
+	seeed-can-fd-hat-v2.dtbo \
 	sh1106-spi.dtbo \
 	smi.dtbo \
 	smi-dev.dtbo \
diff --git a/arch/arm/boot/dts/overlays/README b/arch/arm/boot/dts/overlays/README
index eaaa8a726be0..cc7134913dd3 100644
--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -2504,11 +2504,11 @@ Info:   This overlay is now deprecated. Use the sd_* dtparams in the
 Load:   <Deprecated>
 
 
-Name:   seeed-can-fd-hat
-Info:   Overlay for Seeed Studio CAN BUS FD HAT with two CAN FD channels and an
-        RTC.
+Name:   seeed-can-fd-hat-v2
+Info:   Overlay for Seeed Studio CAN BUS FD HAT with two CAN FD channels
+        (based on the mcp2518fd) and an RTC.
         https://www.seeedstudio.com/CAN-BUS-FD-HAT-for-Raspberry-Pi-p-4742.html
-Load:   dtoverlay=seeed-can-fd-hat
+Load:   dtoverlay=seeed-can-fd-hat-v2
 Params: <None>
 
 
diff --git a/arch/arm/boot/dts/overlays/seeed-can-fd-hat-overlay.dts b/arch/arm/boot/dts/overlays/seeed-can-fd-hat-v2-overlay.dts
similarity index 100%
rename from arch/arm/boot/dts/overlays/seeed-can-fd-hat-overlay.dts
rename to arch/arm/boot/dts/overlays/seeed-can-fd-hat-v2-overlay.dts
-- 
2.17.1

